go/consensus/tendermint/abci/mux.go

```diff
@@ -566,6 +566,12 @@ func (mux *abciMux) BeginBlock(req types.RequestBeginBlock) types.ResponseBeginB
 	return response
 }
 
+func (mux *abciMux) TestExecuteTx(tx []byte) error {
```

Comment:
Where is this used?

Reply:
It used to be used in a few previous incarnations but dead now. Thanks.